### ISSUE TITLE
Feature Request: noise threshold

### ISSUE THREAD

**Issue by Menelmacar** [enguerrand/child-monitor#21]
Hi I stumbled upon this software, and am lucky I found a free software baby monitor.

I didn't use it extensively yet. But the initial setup was already great and it "just worked"

I would like to suggest an option to configure a threshold , so the child device doesn't send constantly, but only after a louder notice was present. 
Currently even the slightest noises are noticeable, making me a bit alert all the time 😂

**Comment by enguerrand** [comment]
However, I personally like having the child device sending constantly so as to be able to detect when it stops sending because it went offline or something like that. (I actually spent some building this in and would view it as a bug if it did not work that way). 
If this is about saving bandwidth then we'd need a parallel keepalive protocol to make sure the child device was still there, but that would increase code complexity and would actually not even guarantee as nicely that the audio stream is really still there.
If this is really about not "alerting" then I think scaling the volume down to zero if it below some threshold would be the more feasible approach. But even this does not make me comfortable, because that threshold is not straight forward to find.
We really want to err on the safe side here, the last thing I'd want to risk is having false positives that mute baby sounds we actually wanted to hear. 
So to make a long story short: Thank you for your suggestion, but I am not willing to implement this and I'd be quite reluctant to even accept a fully working PR that adds this feature, sorry.

**Comment by enguerrand** [comment]
> A "a parallel keepalive protocol" would be needed anyway imho, because that could trigger a warning sound as soon as the connection is broken.

The current implementation sets a timeout of 30 seconds on read operations.
After that timeout, if no audio was streamed, the parent device will close the connection to the child device resulting in the broken connection notification sound.
I don't see how a broken connection could go unnoticed with this implementation.
If however you can identify a scenario where a connection loss does go unnoticed please do open a new issue with step by step instructions to reproduce.
